recipe_batches/recipe_batches.py

```diff
@@ -3,12 +3,42 @@
 import math
 
 def recipe_batches(recipe, ingredients):
-  pass 
+  #Set an empty array
+  values = []
+  #For every index in the given dictionary of recipes
+  for i in recipe:
+        #if that index doesnt exist within ingredients
+        if i not in ingredients:
+              #return 0
+              return 0
+        #Otherwise if it does exist
+        else:
+            print("ingredient amount:", ingredients[i], "\nrecipe amount:", recipe[i])
+            #Append a single value of ingredients divided by recipe
+            values.append(ingredients[i] // recipe[i])
+            #Prints values being appended
+            print("divided value:",values)
 
+  #"smallest" = the first element in vlaues
+  smallest = values[0]
+  #For every index in values
+  for i in range(len(values)):
+      #if that index is less than "smallest" (0)
+      if values[i] < smallest:
+          #Then the "smallest" now equals that value
+          smallest = values[i]
+          
```

Comment:
But if the comments help you understand your code better when you come back to revise it, then keep doing it